src/asm/rgbasm.5

```diff
@@ -860,58 +860,102 @@ code that can be invoked later.
 A text string that can be expanded later, similarly to a macro.
 .El
 .Pp
-Symbol names can contain letters, numbers, underscores
+Symbol names can contain ASCII letters, numbers, underscores
 .Sq _ ,
 hashes
 .Sq #
 and at signs
 .Sq @ .
 However, they must begin with either a letter or an underscore.
-Periods
-.Sq \&.
-are allowed exclusively in labels, as described below.
+Additionally, label names can contain up to a single dot
+.Ql \&. ,
+which may not be the first character.
+.Pp
 A symbol cannot have the same name as a reserved keyword.
-.Ss Label declaration
-One of the assembler's main tasks is to keep track of addresses for you, so you can work with meaningful names instead of "magic" numbers.
+.Ss Labels
+One of the assembler's main tasks is to keep track of addresses for you, so you can work with meaningful names instead of
+.Dq magic
+numbers.
+Labels enable just that: a label ties a name to a specific location within a section.
+A label resolves to a bank and address, determined at the same time as its parent section's (see further in this section).
+.Pp
+A label is defined by writing its name at the beginning of a line, followed by one or two colons,
+.Em without any whitespace between the label name and the colon(s) .
```

Comment:
```suggestion
without any whitespace between the label name and the colon(s) .
```

Reply:
This is my only comment, otherwise it's a well-written explanation. :+1:

Reply:
And here I thought I had fixed it ><